src/nopoll_conn.c

```diff
@@ -882,7 +926,7 @@ noPollConn * __nopoll_conn_new_common (noPollCtx       * ctx,
 	if (session == NOPOLL_INVALID_SOCKET) {
 		/* release connection options */
 		__nopoll_conn_opts_release_if_needed (options);
-		nopoll_log (ctx, NOPOLL_LEVEL_DEBUG, "Failed to connect to remote host %s:%s", host_ip, host_port);
+		nopoll_log (ctx, NOPOLL_LEVEL_CRITICAL, "Failed to connect to remote host %s:%s", host_ip, host_port);
```

Comment:
@bill1600 Is this log level change needed?
We have other logs to indicate the connection host and port.

Reply:
Don't we want to see a log message when connection has failed ?

Reply:
Don' we want to see a log message when the connection fails ?

Reply:
No this log does not give any extra information. All that info is already known in the caller i.e. parodus